ember-cli-build.js

```diff
@@ -103,23 +103,9 @@ function routeRecognizer() {
 }
 
 function router() {
-  // TODO upstream this to router.js and publish on npm
-  return new Rollup('bower_components/router.js/lib', {
+  return new Rollup(path.resolve(path.dirname(require.resolve('router_js')), '../lib'), {
```

Comment:
Can we base this off information in `router_js`' `package.json`?

Something like:
`path.resolve(path.dirName(require.resolve('router_js')), fs.readFileSync(require.resolve('router_js/package.json')).main)`?